indico/client/client.py

```diff
@@ -22,8 +39,18 @@ def _handle_request_chain(self, chain: RequestChain):
 
     def call(self, request: Union[HTTPRequest, RequestChain]):
         """
-        Make a call to the indico IPA platform
+        Make a call to the Indico IPA Platform
+
+        Args:
+            request (HTTPRequest or RequestChain): GraphQL request to send to the Indico Platform
+        
+        Returns:
+            JSON Response of the request
```

Comment:
Not always true, return T depending of the type returned by the request class

Reply:
Changed to...
Returns:
    Response appropriate to the class of the provided request parameter. Often JSON but not 
    always.